docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -145,7 +145,7 @@ Function: `high_mean`.
 
 anomalous_error_rate_for_user_agents_jsbase::
 
-* For APM data where `agent.name` is `js-base`.
+* For data from {apm-rum-agents} (where `agent.name` is `js-base`).
```

Comment:
FYI, we are in the process of renaming the agent name to `rum-js` https://github.com/elastic/apm-agent-rum-js/pull/548/ from the next major version `5.x`

Reply:
@vigneshshanmugam Thanks for the info! I subscribed to the related PR that you linked and will open one for changing the agent name here in the OOTB ML jobs page. After https://github.com/elastic/apm-agent-rum-js/pull/548 is merged, I'll merge mine that changes `js-base` to `rum-js` on this page.